prisma/schema.prisma

```diff
@@ -71,3 +72,38 @@ model VerificationToken {
 
     @@unique([identifier, token])
 }
+
+model Book {
+    id        Int      @id @default(autoincrement())
```

Comment:
let's use String cuids() instead of incrementing IDs. more secure / less error prone